mlrun/api/crud/model_endpoints.py

```diff
@@ -977,43 +666,500 @@ def deploy_model_monitoring_batch_processing(
+
+        :return: List of model endpoints unique ids.
+        """
+
+        # Retrieve the raw data from the KV table and get the endpoint ids
+        cursor = self.client.kv.new_cursor(
+            container=self.container,
+            table_path=self.path,
+            filter_expression=self.build_kv_cursor_filter_expression(
+                self.project,
+                function,
+                model,
+                labels,
+                top_level,
+            ),
+            attribute_names=["endpoint_id"],
+            raise_for_status=v3io.dataplane.RaiseForStatus.never,
+        )
+        try:
+            items = cursor.all()
+        except Exception:
+            return []
+
+        # Create a list of model endpoints unique ids
+        uids = [item["endpoint_id"] for item in items]
+
+        return uids
+
+    def delete_model_endpoints_resources(
+        self, endpoints: mlrun.api.schemas.model_endpoints.ModelEndpointList
+    ):
+        """
+        Delete all model endpoints resources in both KV and the time series DB.
+
+        :param endpoints: An object of ModelEndpointList which is literally a list of model endpoints along with some
+                          metadata. To get a standard list of model endpoints use ModelEndpointList.endpoints.
+        """
+
+        # Delete model endpoint record from KV table
+        for endpoint in endpoints.endpoints:
+            self.delete_model_endpoint(
+                endpoint.metadata.uid,
+            )
+
+        # Delete remain records in the KV
+        all_records = self.client.kv.new_cursor(
+            container=self.container,
+            table_path=self.path,
+            raise_for_status=v3io.dataplane.RaiseForStatus.never,
+        ).all()
+
+        all_records = [r["__name"] for r in all_records]
+
+        # Cleanup KV
+        for record in all_records:
+            self.client.kv.delete(
+                container=self.container,
+                table_path=self.path,
+                key=record,
+                raise_for_status=v3io.dataplane.RaiseForStatus.never,
+            )
+
+        # Cleanup TSDB
+        frames = mlrun.utils.v3io_clients.get_frames_client(
+            token=self.access_key,
+            address=mlrun.mlconf.v3io_framesd,
+            container=self.container,
+        )
+
+        # Getting the path for the time series DB
+        events_path = (
+            mlrun.mlconf.model_endpoint_monitoring.store_prefixes.default.format(
+                project=self.project,
+                kind=mlrun.api.schemas.ModelMonitoringStoreKinds.EVENTS,
+            )
+        )
+        (
+            _,
+            _,
+            events_path,
+        ) = mlrun.utils.model_monitoring.parse_model_endpoint_store_prefix(events_path)
+
+        # Delete time series DB resources
+        try:
+            frames.delete(
+                backend="tsdb",
```

Comment:
would change it to a static variable at the beginning of the file.

Reply:
added under `mlrun.model_monitoring.constants`:
```
class TimeSeriesTarget:
    TSDB = "tsdb"
```

And in the new code:
```
try:
    frames.delete(
        backend=model_monitoring_constants.TimeSeriesTarget.TSDB,
        table=events_path,
        if_missing=v3io_frames.frames_pb2.IGNORE,
    )
```